orchagent/portsorch.cpp

```diff
@@ -7345,6 +7521,18 @@ void PortsOrch::doTask(NotificationConsumer &consumer)
 
         sai_deserialize_free_port_oper_status_ntf(count, portoperstatus);
     }
+    else if (&consumer == m_portHostTxReadyNotificationConsumer && op == "port_host_tx_ready")
```

Comment:
@noaOrMlnx Have you tested, when user does admin down, the host_tx_ready is false?

Reply:
Yes